### PR TITLE
Updated the Content of Polyfills Page of DSA in JS

### PR DESCRIPTION
**Pull request by Abhish7k**
## Changes proposed
Added the content of the Polyfills page of DSA in JS.
Also corrected a grammatical error that was present in every content page. There was just a full stop after "?" that I removed.

## Check List

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Content added:
Screenshot (48)

Grammatical mistake corrected:

before:
Screenshot (50)

after:
Screenshot (49)

### CODE REVIEW

File: package-lock.json
Comment (ManishBisht777) [inline package-lock.json:1]: Make sure to remove lock file

File: content/batch/build/interview/counter.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: Interview Question
 
 <Callout>
 
-Want to improve this page?. Raise a issue on [@github](https://github.com/ManishBisht777/frontend-freaks-website).
+Want to improve this page? Raise a issue on [@github](https://github.com/ManishBisht777/frontend-freaks-website).
```
Comment (ManishBisht777) [inline content/batch/build/interview/counter.mdx:8]: Why these many changes are there ?

**Review by ManishBisht777 — COMMENTED** [review]
Left some comments

File: package-lock.json
Comment (Abhish7k) [reply package-lock.json:1]: Ok, I will remove it.

File: content/batch/build/interview/counter.mdx (hunk 1 above)
Comment (Abhish7k) [reply content/batch/build/interview/counter.mdx:8]: These were the grammatical errors on every content page. Should I include them in a separate pr ?

Comment (ManishBisht777) [reply content/batch/build/interview/counter.mdx:8]: let's keep them since you already solved it

File: package-lock.json
Comment (ManishBisht777) [reply package-lock.json:1]: remove it from pr, not from the code itself and try checking the build it on local first there are build errors can you solve it?

Comment (Abhish7k) [reply package-lock.json:1]: ok, working on it, will try to solve them.

File: content/batch/build/interview/counter.mdx (hunk 1 above)
Comment (Abhish7k) [reply content/batch/build/interview/counter.mdx:8]: Thank you !

File: package-lock.json
Comment (Abhish7k) [reply package-lock.json:1]: The build fails only when I remove the package-lock file, build works fine on local too.

**Pull request merged**